README.md

```diff
@@ -26,35 +26,41 @@ supported in Windows):
 On Ubuntu:
 
     ./bootstrap.sh
-    
+
 This will install Anaconda with Python 3, which includes dependent
-libraries such as scikit-learn. 
-    
+libraries such as scikit-learn.
+
 On MacOS:
 
     brew install qt
     brew install python3
     pip3 install -U numpy scipy scikit-learn
     pip3 install -r requirements.txt
 
-    
+
 If you already have Python 3, scikit-learn,
 NumPy and SciPy install you will only need to do
 
     pip install -r requirements.txt
 
+You will also need to gect the nltk corpora data:
+
+    python - <<EOF
+    import nltk
+    nltk.download('all-corpora')
```

Comment:
We can add to bootstrap.sh the following lines:

```
python -m nltk.downloader punkt
python -m nltk.downloader stopwords
python -m nltk.downloader wordnet
```
rather than downloading the entire corpus. This will also set up the environment in one script rather than having to run separate statements.